hack/sonobuoy/cyclonus-plugin.yaml

```diff
@@ -1,15 +1,16 @@
 sonobuoy-config:
   driver: Job
   plugin-name: cyclonus
-  result-format: raw
+  result-format: junit
 spec:
   command:
   - ./run-sonobuoy-plugin.sh
   - generate
   - "--include=conflict"
   - "--exclude=egress,direction"
+  - "--junit=/tmp/results/junit.xml"
   image: mfenwick100/sonobuoy-cyclonus:latest
-  imagePullPolicy: IfNotPresent
+  imagePullPolicy: Always
```

Comment:
Common issue, especially if you are working on developing a plugin, is that you dont pull in new versions as you're developing. Switching this to Always helps fix that problem and wont cause anyone too much trouble.

Reply:
👍 🚀